README.md

```diff
@@ -1,280 +1,231 @@
+```
 
-### Without Docker
+If you are a validator, this sequence of commands will supply Parity with your keystore file, password, and address so
+that it can produce blocks:
 
-This section explains how to start a local peer without using the _Docker_ image.
+```sh
+$ mkdir -p trustlines/databases trustlines/config/keys/Trustlines trustlines/enode trustlines/shared
+$ cp /path/to/your/keystore/file.json trustlines/config/keys/Trustlines
+$ echo "<passphrase_for_keystore_file>" > trustlines/config/pass.pwd
+$ docker run -d --name laika-node --network network-laika -v $(pwd)/trustlines/databases:/data -v $(pwd)/trustlines/config:/config/custom -v $(pwd)/trustlines/enode:/config/network -v $(pwd)/trustlines/shared:/shared/ -p 30300:30300 -p 30300:30300/udp trustlines/tlbc-testnet --role validator --address MY_ADDRESS
+```
 
-#### Pre-Requisites
+#### Netstats Client
 
-- parity version 2.0.9
-- the file: "trustlines-spec.json"
+The netstats client reports the state of your node to [https://laikanetstats.trustlines.foundation/]() that gives a
+rough overview over the current state of the network. It is a fully optional component which helps the community, at
+the cost leaks of some leakage of private information.
 
-#### Creating An Account
+To participate, you first need to request credentials managed by the Trustlines Foundation. Please email
+`netstats@trustlines.foundation` to do so.
 
-To interact with the trustlines chain, one needs a private key corresponding to an address usable on the chain. If you already possess such a key, you can skip this section.
+Next, create a file named `netstats-env` in the `trustlines` directory with the following contents:
 
-To start with, create a folder to store everything related to the Trustlines chain, move the trustlines-spec.json file to this folder and change to this folder:
+```sh
+WS_USER=username-as-provided-by-the-foundation
+WS_PASSWORD=password-as-provided-by-the-foundation
+INSTANCE_NAME=please-choose-a-nice-name-here
+```
+
+If you want to be publicly reported as a validator, add the following line at the end:
 
 ```sh
-mkdir trustlines-chain
-mv trustlines-spec.json trustlines-chain/trustlines-spec.json
-cd trustlines-chain
+HIDE_VALIDATOR_STATUS=false
 ```
 
-You can then create an account with the command:
+The netstats client has to connect to the blockchain node. Therefore, we have to find out the node's IP address with
 
 ```sh
-parity account new --chain trustlines-spec.json -d [path/to/node/foler]
+docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' laika-node
 ```
 
-For the rest of this documentation to become a validator we will assume you ran:
+Now, the netstats client can be started with
 
 ```sh
-parity account new --chain trustlines-spec.json -d validator_node
+$ docker run --name netstats-client --network network-laika -e trustlines/netstats-env -e RPC_HOST=<node IP address> -e RPC_PORT=8545 trustlines/netstats-client:master
```

Comment:
This command doesn't work. The netstats client connects to the node just fine via HTTP, but then apparently tries to make a websocket request which fails. Which is weird, because as far as I do pretty much the same as the docker compose file.

Reply:
Now it seems to not even be able to make the HTTP request.